apps/ledger-live-desktop/src/renderer/modals/RequestAccount/Body.jsx

```diff
@@ -19,7 +19,7 @@ type OwnProps = {|
   params: {
     currencies?: string[],
     allowAddAccount?: boolean,
-    onResult: (account: AccountLike, parentAccount?: Account | null) => void,
+    onResult: (account?: AccountLike, parentAccount?: Account | null) => void,
```

Comment:
Same here, account should be mandatory IMO

Reply:
It's a side effect of `SelectAccountAndCurrency::selectAccount` method signature.